### ISSUE TITLE
Effect of not using nuclear corrections

### ISSUE THREAD

**Issue by aaccardi** [JeffersonLab/CJ#5]
Results of the 'no nuclear corrections' series of fits  described in __section (D) of the [README_LIST_OF_FITS](https://github.com/JeffersonLab/CJ/blob/master/CJ15PLOTS/CJ15-fits/README_LIST_OF_FITS)__

__Goal__

The goal is to prove that it is the tension between SLAC(D) and DO Wasy data that allows one to fit the nuclear corrections, and thus reduces the nculear uncertainty that afflicted CJ12. In fact, this would not have been possible then with only CDF Wasy data.

__Results__

The table below summarizes the &chi;<sup>2</sup> per data set for a series of fits that did not include nuclear corrections, and progressively including the d-quark sensitive data sets in the headers, starting from 'DIS' that has all inclusive DIS sets, DY, and Jets. The last column show the AV18 fit, where the nuclear corrections have been turned on.

<pre>
<b>data set    npts     DIS     BNS    Lasy    Zrap    WCDF     WD0    AV18</b>

jl00106F2p   136   166.0   165.6   165.6   165.7   166.1   176.7   165.8
jl00106F2d   136   132.2   130.1   130.1   131.1   130.3   126.0   122.7
HerF2pCut     37    44.4    44.1    44.1    44.2    44.3    44.0    41.9
HerF2dCut     37    37.3    38.1    38.1    37.7    37.4    35.9    36.6
slac_p       564   437.4   442.1   442.1   439.0   439.9   441.3   433.6
<b>slac_d       582   471.0   460.9   460.9   467.4   466.4   507.3   376.3</b>
BcdF2pCor    351   429.7   425.5   425.5   427.7   426.8   439.6   437.5
BcdF2dCor    254   305.1   303.1   303.1   303.6   301.0   301.0   292.3
NmcF2pCor    275   400.9   401.1   401.1   401.7   403.2   405.2   404.6
NmcRatCor    189   173.4   176.3   176.3   173.1   172.9   173.5   172.2
H2_NC_em     159   243.8   244.1   244.1   243.8   243.9   246.9   241.0
H2_NC_ep_1   402   582.9   581.7   581.7   582.7   584.6   587.8   579.7
H2_NC_ep_2    75    94.1    94.0    94.0    93.8    93.1    93.7    94.1
H2_NC_ep_3   259   248.4   248.3   248.3   247.9   247.6   247.8   248.7
H2_NC_ep_4   209   228.8   228.8   228.8   228.5   227.7   227.8   228.1
H2_CC_em      42    47.4    49.5    49.5    48.4    48.7    45.2    47.8
H2_CC_ep      39    49.9    50.6    50.6    50.2    50.7    50.9    50.1
BNS_F2nd     191           216.5   216.5   218.3   218.9   219.2   214.2
e866pp06xf   121   142.2   143.3   143.3   142.6   143.4   144.7   138.8
e866pd06xf   129   152.5   155.4   155.4   155.8   157.1   157.7   144.8
cdfLasy05     11                            13.5    13.5    12.9    12.2
d0Lasy_e15    13                            30.9    28.2    29.2    28.6
d0Lasy13      10                            12.4    13.8    13.5    19.2
CDF_Wasy      13                                    14.4    14.1    16.3
<b>D0_Wasy       14                                            81.9    13.5</b>
CDF_Z         28                                    26.0    25.6    27.3
D0_Z          28                                    15.6    15.7    15.8
d0run2cone   110    25.2    25.4    25.4    24.9    25.1    22.7    20.7
CDFrun2jet    72    14.5    14.5    14.5    14.4    14.4    14.1    14.9
d0_gamjet1    16     6.8     6.9     6.9     6.9     7.0     7.1     6.5
d0_gamjet2    16    16.5    16.6    16.6    16.8    17.0    17.1    16.2
d0_gamjet3    12    24.8    24.9    24.9    24.7    24.7    24.7    25.2
d0_gamjet4    12    12.8    12.7    12.7    12.8    12.9    13.0    13.1
------------------------------------------------------------------------
TOTAL       4542  4487.7  4700.2  4700.2  4760.6  4816.7  4963.8  4700.2
<b>TOTAL+norm        4495.0  4709.0  4709.0  4768.8  4825.6  4972.0  4707.9</b>
no. pts             4234    4425    4425    4459    4528    4542    4542
(TOT+nrm)/pts       1.06    1.06    1.06    1.07    1.07    1.09    1.04
</pre>

One notices that:
* Until one includes D0 Wasy data, no DIS(D) data sets is in clear tension with any d-quark sensitive set. 
** This is seen by the (relative) stability of the &chi;<sup>2</sup> of the DIS(D) sets as more and more data is included in the fit.
* After including the D0 Wasy data, the SLAC(D) &chi;<sup>2</sup> jumps up by 40 units, indicating clear disagreement with D0. The D0 Wasy has chi^/pt~5.8, that shows strong tension with SLAC(D).
** Other sets also go up and down a bit after including D0 Wasy, but never more than 10 units
** Other sets, such as Zrap seem to have a mild effect on SLAC(D), but overall it seems that it is DO Wasy that unambiguously pulls against SLAC(D)
** Other DIS(D) data sets do not seem to be much affected (Either due to snall x covergae such as JLab(D) or too large errors such as NMC)
* Finally, switching on the nuclear corrections keeping the data set fixed ('AV18' fit) brings down the total &chi;<sup>2</sup> by 273 units, most of which due to SLAC(D) and D0 WAsy, that decreases by 131 and 70 units, respectively
** This shows that w/o nuclear corrections the 2 data sets are in clear tension, and the fit tries to accommodate both with an intermediate d-quark that, however, doesn't fit well either set.
** There are also minor, O(10) contributions from a variety of sets including DIS p & D, and DY. This shows that not using nuclear corrections affects not only the large x regime, but the data do not clearly indicate a need for these until DO Wasy is included.

__Conclusions__

It is the interplay between D0 Wasy and SLAC(D) that allows one to fit the nuclear corrections and reduces the theoretical nuclear uncertainty that was estimated in CJ12.

In summary (see also  issue #2 ):
* D0 Wasy data at x>~0.3 can constrain the large-x d-quark and provide the largest reduction in relative error
* DIS(D) data play a double role:
** at large x>~0.3, thanks mainly to  SLAC(D), they reduce the theoretical nuclear uncertainty, and help further reduce teh statistical uncertainty
** at small x < 0.3 they provide strong constraints on the d-quark statistical uncertainties, that are not possible using DO Wasy.
